### ISSUE TITLE
Is image features from frozen Resnet152 or trainable resnet152? Seems frozen in code and trainable in paper. 

### ISSUE THREAD

**Issue by SunTongtongtong** [Cuberick-Orion/CIRPLANT#8]
Hi there,

I notice in the code command you use `--usefeat nlvr-resnet152_w_empty` to specify the image features are from ImageNet pretrained ResNet152. I redo this and do achieve the reported result as paper. However, I have a question as I also notice in paper you mention:
> For our proposed model, we use ResNet152 for image feature extraction. The model is optimized with AdamW [27] with an initial learning rate of 10−5.

,which means you use a learnable resnet152.

I'm confused by this. 
Can you confirm which is correct?

Best

**Comment by Cuberick-Orion** [comment]
Hi,

Apologies for the confusion -- that is a wording issue. "The model" suggests our model (i.e., model based on OSCAR), not the ResNet.

We do not fine-tune ResNet152 in this case.
